Source/WebCore/layout/formattingContexts/inline/InlineItem.cpp

```diff
@@ -35,6 +35,7 @@ namespace Layout {
 struct SameSizeAsInlineItem {
     void* layoutBox;
     float width;
+    float offset;
```

Comment:
this is quite an expansion for something this uncommon.

Reply:
I actually could use `width` the width of InlineItem (I guess it is not used when the item is a `Type::InlineBoxStart` ?) to save a float, but `width` does not hold the right meaning of what we want to do. Do you think it would be better, or should we just find a better name for `offset`?

Reply:
inline boxes can have decoration(s) so width is actually used on them. I can think of 2 alternatives here.
- use a side structure for this data
- keep it part of the InlineTextItem width and transfer it over to outside of the inline box while building the line (but this is probably too involved and error prone)